index.html

```diff
@@ -10,6 +10,72 @@
 </head>
 
 <body>
+    <main>
+        <div class='container'>
+            <div class="filler"></div>
+            <nav class='navbar'>
+                <a href=""><img src="images/Apple-Logo.png" alt=""></a>
```

Comment:
The `alt` attribute provide alternative text if the image for any reason can't be displayed, so let's fill it with proper description.